client/src/components/CustomButton.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { MouseEventHandler, ReactNode } from "react";
+import { Button } from "@mui/material";
+import { makeStyles } from "@mui/styles";
+
+interface Props {
+  children: ReactNode;
+  onClick?: MouseEventHandler<HTMLButtonElement>;
+  color?:
+    | "inherit"
+    | "primary"
+    | "secondary"
+    | "success"
+    | "error"
+    | "info"
+    | "warning";
+  variant?: "text" | "outlined" | "contained";
+  disabled?: boolean | undefined;
+}
+
+export const CustomButton = ({
+  children,
+  onClick,
+  color = "primary",
+  variant = "contained",
+  disabled = false,
+}: Props) => {
+  return (
+    <Button
+      sx={{
+        borderRadius: "3.75em",
```

Comment:
I am wondering if this might be a little more readable if we pull it out into a style object.

But, it works, so I'm also happy leaving it here!

Reply:
Good call. Just updated the PR and also pulled out styles on the Grid in selection.tsx :)